### PR TITLE
Add Certificate Helper for Developing on Linux (Debian)

### PR DESCRIPTION
**Pull request by justindbaur**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other
```

## Objective

I am looking to do more of my development on WSL Ubuntu and this is the main thing keeping me from getting my dev environment working. This will make it so that `Identity` will be able to find the dev certificate on my machine. The environment this was explicitly tested for was: 
```shell
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.1 LTS
Release:        22.04
Codename:       jammy
```

## Code changes

* **dev/create_certificates_linux.sh:** Creates a `dash` compatible shell script to be able to generate and install valid certificates on a debian based linux install that might not even have bash installed. 

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: dev/create_certificates_linux.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+dotnet tool install -g dotnet-certificate-tool
+
+stty -echo
+printf "Identity Server Dev Password: "
```
Comment (MGibson1) [inline dev/create_certificates_linux.sh:6]: clarify `(enter for no password)`

Comment (MGibson1) [inline dev/create_certificates_linux.sh:6]:
```suggestion
printf "Identity Server Dev Password: (enter for no password)"
```

File: dev/create_certificates_linux.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    -subj "/CN=Bitwarden Identity Server Dev" -days 3650
+openssl pkcs12 -export -out identity_server_dev.pfx -inkey identity_server_dev.key -in identity_server_dev.crt \
+    -certfile identity_server_dev.crt -password pass:$identity_server_dev_pass
+
+identity="$(openssl x509 -in identity_server_dev.crt -outform der | shasum -a 1 | head -c 40 | tr a-z A-Z)"
+
+dotnet certificate-tool add --file ./identity_server_dev.pfx --password $identity_server_dev_pass
+
+stty -echo
+printf "Bitwarden Data Protection Dev Password:"
```
Comment (MGibson1) [inline dev/create_certificates_linux.sh:21]:
```suggestion
printf "Bitwarden Data Protection Dev Password: (enter for no password)"
```

File: dev/create_certificates_linux.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+printf "\n"
+
+openssl req -x509 -newkey rsa:4096 -sha256 -nodes -keyout identity_server_dev.key -out identity_server_dev.crt \
+    -subj "/CN=Bitwarden Identity Server Dev" -days 3650
+openssl pkcs12 -export -out identity_server_dev.pfx -inkey identity_server_dev.key -in identity_server_dev.crt \
+    -certfile identity_server_dev.crt -password pass:$identity_server_dev_pass
+
+identity="$(openssl x509 -in identity_server_dev.crt -outform der | shasum -a 1 | head -c 40 | tr a-z A-Z)"
+
+dotnet certificate-tool add --file ./identity_server_dev.pfx --password $identity_server_dev_pass
```
Comment (MGibson1) [inline dev/create_certificates_linux.sh:18]: did you confirm the behavior of this if no password is given?

Comment (justindbaur) [reply dev/create_certificates_linux.sh]: I have now and had to add an if else for no password but it works.

**Comment by justindbaur** [comment]
Closing in favor of bringing #2941 one to the finish line

**Pull request closed without merging**